podcasts/PlaybackActionHelper.swift

```diff
@@ -7,6 +7,8 @@ class PlaybackActionHelper {
     class func play(episode: BaseEpisode, filterUuid: String? = nil, podcastUuid: String? = nil) {
         HapticsHelper.triggerPlayPauseHaptic()
 
+        AutoplayHelper.shared.savePlaylist()
```

Comment:
I don't like that this is a singleton. I pretend to change that once I make `PlaybackActionHelper` a singleton (it's a collection of static methods right now).

Refactoring this would skyrocket the size of this PR.